src/app/MenuView.ts

```diff
@@ -14,92 +15,63 @@ export class MenuView {
 
   displayMenu(): void {
     Render.removeAllChildren('#sf-app');
-    Render.render(
-      '#sf-app',
-      this.createLandingPage(),
-      this.createFooter(),
-    );
+    Render.render('#sf-app', this.createLandingPage());
   }
 
   private createLandingPage(): HTMLElement {
     return Render.elementFactory(
       'div',
       {
-        className: 'page__container menu-window',
+        className: 'page__container',
       },
-      this.createHeading(),
-      this.createPageContent(),
+      Render.elementFactory(
+        'div',
+        { className: 'menu-window' },
+        this.createHeading(),
+        this.createPageContent(),
+        this.createFooter(),
+      ),
+      this.createStartButton(),
```

Comment:
Solution for not breaking the layout, would be to keep everything in one container (so create start button inside the same menu window, just before footer)